### PR TITLE
Using service-print as print backend

### PR DESCRIPTION
**Pull request by procrastinatio**
This PR makes the print backend configurable. The goal being to isolate the print service from `mf-chsdi3`

A new environmental variable `PRINT_URL`is defined, being set by default to `//service-print.dev.bgdi.ch`

Please review, and event. merge and deploy https://github.com/geoadmin/service-print/pull/4 before anything. 
You may already test against [dev](//service-print.dev.bgdi.ch/print/info.json) and [int](//service-print.int.bgdi.ch/print/info.json)

[Test link](https://mf-geoadmin3.dev.bgdi.ch/mom_using_service_print/) for geoadmin3

The service urls are:
* service-print.dev.bgdi.ch
* service-print.int.bgdi.ch
* service-print.prod.bgdi.ch
* print.geo.admin.ch.

See https://github.com/geoadmin/service-print/pull/4 for backend (merged)

### CODE REVIEW

**Comment by procrastinatio** [comment]
@gjn @loicgasser You may tests and review.

`service-print` deploys well on `dev`and `int` from snapshot.

**Comment by loicgasser** [comment]
jenkins test

**Comment by loicgasser** [comment]
Works as expected and LGTM

Did you redeploy using the scripts and snapshots or is it the old back-end version?

**Comment by loicgasser** [comment]
Maybe add PRINT_URL in https://github.com/geoadmin/mf-geoadmin3/blob/master/rc_user

so that we can test `print_multi.py`

**Comment by gjn** [comment]
We also need a print for the `ci` environment. Seleniums tests are run in ci. Either we add the vhost in ci as well or we use the dev enviorment for ci print.

File: Makefile [hunk 1]
```diff
@@ -346,6 +349,7 @@ prd/geoadmin.appcache: src/geoadmin.mako.appcache \
 	    --var "apache_base_path=$(APACHE_BASE_PATH)" \
 	    --var "languages=$(LANGUAGES)" \
 	    --var "api_url=$(API_URL)" \
+		--var "print_url=$(PRINT_URL)" \
```
Comment (loicgasser) [inline Makefile]: indent

File: Makefile [hunk 2]
```diff
@@ -497,6 +502,7 @@ apache/app.conf: apache/app.mako-dot-conf \
 	${PYTHON_CMD} ${MAKO_CMD} \
 	    --var "apache_base_path=$(APACHE_BASE_PATH)" \
 	    --var "api_url=$(API_URL)" \
+		--var "print_url=$(PRINT_URL)" \
```
Comment (loicgasser) [inline Makefile]: indent

**Comment by loicgasser** [comment]
I rebased and fixed.

**Comment by loicgasser** [comment]
For me this is good to go and everything is ready to separate the print.

I deployed the print using a snapshot withthin the service-print project and deployed this branch to int. Here is a [Test Link](https://mf-geoadmin3.int.bgdi.ch/gal_using_service_print/?topic=ech&lang=fr&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.swisstopo.zeitreihen,ch.bfs.gebaeude_wohnungs_register,ch.bav.haltestellen-oev,ch.swisstopo.swisstlm3d-wanderwege&layers_visibility=false,false,false,true&layers_timestamp=18641231,,,&X=224475.53&Y=599043.14&zoom=5)

**Pull request closed without merging**